### PR TITLE
DG-8155: Fix alpha channels

### PR DESCRIPTION
**Pull request by richardadjogah**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -31,17 +31,18 @@ if (cluster.isMaster) {
...
 
 
-	app.get("/favicon.ico", function(req, res) {
+	app.get('/favicon.ico', function(req, res) {
 		res.status(200).send();
 		// Eat favicon requests from the browser
 	});
 
 	app.get(/.*/, function(req, res) {
+		res.header('Access-Control-Allow-Origin', '*');
```
Comment (nchase) [inline app.js]: :clap: – we might want to control this a bit more finely-grained some time, but this is great :clap:

File: test/imageHelperTest.js [hunk 2]
```diff
@@ -9,20 +9,32 @@ describe('Image Helper Test', function () {
 		done();
 	});
 
-	it('correctly detects image extensions: bmp -> jpeg', function (done) {
+	it('correctly detects image extensions: bmp -> png', function (done) {
```
Comment (nchase) [inline test/imageHelperTest.js]: is there something about the extensions we're checking for in these tests that makes them important (e.g. code or business logic specifically for these file extensions – bmp, gif, mp3, webp, jpeg), or can we just have one test for one arbitrary extension?

File: app.js (hunk 1 above)
Comment (richardadjogah) [inline app.js]: It was originally restricted to `localhost:12000`, but I couldn't think of any reason to restrict it to anyone.

File: test/imageHelperTest.js (hunk 2 above)
Comment (richardadjogah) [inline test/imageHelperTest.js]: `webp`, `gif``, `jp(e)g`, and `png` are the officially supported formats. `mp3` is simply used as an example of an invalid format.

**Comment by twisterghost** [comment]
This specific use case works, though (obviously) the `/jpeg` endpoint still gives us a jpeg, which means no alpha channel. Is there a way we could use `png` compression algorithms to accomplish something similar to the `jpeg` endpoint to create user flows that never have black alpha channels?

**Comment by richardadjogah** [comment]
PNG compression is already maxed out. PNG Filesizes can't get the the same level that JPEGs can simply because PNGs are lossless.

**Comment by twisterghost** [comment]
:shipit:

**Pull request merged**